src/app/clusters/operational-credentials-server/operational-credentials-cluster.cpp

```diff
@@ -95,37 +91,84 @@ CHIP_ERROR CreateAccessControlEntryForNewFabricAdministrator(const Access::Subje
         return err;
     }
 
-    ChipLogProgress(Zcl, "OpCreds: ACL entry created for Fabric index 0x%x CASE Admin Subject 0x" ChipLogFormatX64,
-                    static_cast<unsigned>(fabricIndex), ChipLogValueX64(subject));
+    ChipLogDetail(Zcl, "OpCreds: ACL entry created for Fabric index 0x%x CASE Admin Subject 0x" ChipLogFormatX64,
```

Comment:
The switch from Progress to Detail seems to create a large delta and instability in bloaty. I think changes for this should be separated out.